src/journal/issue_forms.py

```diff
@@ -38,3 +38,20 @@ class IssueGalleyForm(core_forms.FileUploadForm):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, mimetypes=self.MIMETYPES, **kwargs)
+
+
+class SortForm(forms.Form):
+    sort_field = forms.ChoiceField(
+        choices=(
+            ('date_published', 'Date Published'),
+            ('title', 'Title, Alphabetically'),
+            ('article_number', 'Article Number'),
+            ('page_numbers', 'Page Numbers'),
+        )
+    )
```

Comment:
This will be super useful, but I wish it took into account the `first_page` and `last_page` fields, since those are the main page number fields (`page_numbers` is just for custom page ranges). What if we add just the `first_page` field and add `(Custom)` to the `Page Numbers` choice label?